java/ql/lib/semmle/code/java/dataflow/internal/DataFlowImpl.qll

```diff
@@ -2920,6 +2938,22 @@ abstract private class PathNodeImpl extends TPathNode {
     )
   }
 
+  string getSourceGroup() {
```

Comment:
Similarly: Should this be `getASourceGroup`?

Reply:
Resolved [here](https://github.com/github/codeql/pull/11183#discussion_r1023853746).